src/mlpack/bindings/python/setup.py.in

```diff
@@ -73,7 +76,9 @@ else:
                   include_dirs=[ \
                       np.get_include(), \
                       '${OUTPUT_DIR}/src/mlpack/bindings/python/'] +
-                      '${CYTHON_INCLUDE_DIRECTORIES}'.split(' '),
+                      '${CYTHON_INCLUDE_DIRECTORIES}'.replace(' ', ';')
```

Comment:
Hm, this only works for a single space, right?

Reply:
I don't understand, why would it only work for a single space? Python's replace will replace all instances, and all directory spaces should be properly escaped.

Reply:
Not `replace` but the construct, I haven't tested it on multiple spaces, so maybe it works.

Reply:
It looks like you're right. `[ 11%] Configuring setup.py...
-- Run with /usr/local/lib/libboost_serialization-mt.dylib.
CMake Error: File /Users/ryanbirmingham/Desktop/git/ml/ / / / / pa/ / / / / / / / / c/ / k/src/mlpack/bindings/python/setup.py.in does not exist.
CMake Error at /Users/ryanbirmingham/Desktop/git/ml     pa         c  k/src/mlpack/bindings/python/ConfigureSetup.cmake:46 (configure_file):
  configure_file Problem configuring file`

Reply:
Hmm, that error doesn't actually look like it's caused by the line here but instead by strange/improper handling at the CMake level.  @birm if you want to take a shot at this feel free---I was going to try to find time for it later tonight but if you think you can get it then I will go review other PRs instead. :) (And if you'd prefer not to that's fine too of course!)

Reply:
I had the wrong branch. I'll do what I can here, starting with testing on the right code.